config/locales/gds_start.en.yml

```diff
@@ -2,25 +2,17 @@
 en:
   pages:
     gds_start_page:
-      heading: Register your flood risk activity exemptions
-      div_text16: 'Telephone: 03708 506 506'
-      div_text17: 'Minicom: 03702 422 549'
-      div_text18: Monday to Friday, 8am to 6pm
-      list_item: Subsection
-      list_item1: Read the
-      list_item2: "."
-      list_item3: Decide whether any of the exemptions apply to your activities.
-      list_item4: Check that each activity will meet the conditions of the relevant
-        exemption.
-      list_item5: Call us to check if your activities are on a main river and if they are near any environmentally
-        sensitive areas or a flood defence. Our number is 03708 506 506.
-      list_item6: the grid reference where the work will happen
-      list_item7: the name and address of the organisation or person who is legally responsible for the work
-      list_item8: for a company, its registration number and registered address
-      list_item9: for a partnership, the name and address of all partners
-      list_item10: contact details for the person we should contact about the work
-      dredging_header: Dredging exemptions
-      dredging_text: "You can’t register the exemptions for dredging, FRA23 and FRA24, online.
-        Please contact us to register these."
-      para_text: It’s free to register an exemption. It should take about 10 minutes.
-      para_text1: 'You''ll need:'
+      heading: Register a flood risk activity exemption
+      before_you_register: 'Before you register you must:'
+      requirement1: check you can meet the description and conditions for the exemption
+      requirement2: have authority to register for an organisation or as an agent, if you are registering on their behalf
+      we_will: 'We will:'
+      responsibility1: check if the location is environmentally sensitive or if it is near a flood defence structure operated by us
+      responsibility2: tell you whether we have registered the exemption within 10 working days
+      you_will_need: 'You will need:'
+      need1: the grid reference where the work will happen
+      need2: the name and address of the organisation or person who is responsible for the work
+      need3: for a company, its registration number and registered address
+      need4: for a partnership, the name and address of all partners
+      need5: contact details for the person we should contact about the work
```

Comment:
If you are going to refactor these, I'd suggest using meaningful names. For example:
```yaml
needs:
  grid_reference: the grid reference where the work will happen exemption.
  name_and_address: the name and address of the organisation or person who is responsible for the work
  registration_details: for a company, its registration number and registered address
  partners_details: for a partnership, the name and address of all partners
  contact_details: contact details for the person we should contact about the work
```